tests/api_connexion/endpoints/test_config_endpoint.py

```diff
@@ -105,6 +105,23 @@ def test_should_respond_200_application_json(self, mock_as_dict):
         }
         assert expected == response.json
 
+    @patch("airflow.api_connexion.endpoints.config_endpoint.conf.as_dict", return_value=MOCK_CONF)
+    def test_should_respond_200_single_section_as_text_plain(self, mock_as_dict):
```

Comment:
Can you add test for when mime type is application/json

Reply:
Added test for `application/json`